### PR TITLE
Added support for multiple character separators at string humanizer

### PR DESCRIPTION
**Pull request by drgomesp**
Regarding #45, now you can pass a special character separator to humanize strings:

```php
$this->humanize('news-count', true, Humanize::SEPARATOR_DASH)->shouldReturn('News count');
```

### CODE REVIEW

File: src/Coduo/PHPHumanizer/String/Humanize.php [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 class Humanize
 {
+    const SEPARATOR_DASH = '-';
+    const SEPARATOR_UNDERSCORE = '_';
```
Comment (norzechowicz) [inline src/Coduo/PHPHumanizer/String/Humanize.php]: I dont think we really need those constants. 
``public function __construct($text, $capitalize = true, $separator = '-')``
should be enough. In next release I would like to change this behavior anyway, I would like to not pass aything by default as a separator.

**Comment by norzechowicz** [comment]
@drgomesp thanks a lot! Please take a look at my comment and I gonna merge it.

File: src/Coduo/PHPHumanizer/String/Humanize.php (hunk 1 above)
Comment (drgomesp) [inline src/Coduo/PHPHumanizer/String/Humanize.php]: Agreed. Just making the change and we will be good to go.

**Comment by drgomesp** [comment]
@norzechowicz :+1:

**Comment by norzechowicz** [comment]
@drgomesp thanks! :beers:

**Pull request merged**